lightningd/options.c

```diff
@@ -163,7 +163,14 @@ static char *opt_add_announce_addr(const char *arg, struct lightningd *ld)
 {
 	const struct wireaddr *wn;
 	size_t n = tal_count(ld->proposed_wireaddr);
-	char *err = opt_add_addr_withtype(arg, ld, ADDR_ANNOUNCE, false);
+	char *err;
+
+	/* Check for autotor and reroute the call to --addr  */
+	if (strstarts(arg, "autotor:"))	{
```

Comment:
Nitpick, for `if` with one statement we omit `{ }`.

Reply:
Done :-) and rebased 

The rebase to current master needs the opt-add-addr declaration to be on top of source so 
please review again.

Reply:
Looks OK, suggest squashing into one commit.

Reply:
Done :-)